Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1606,9 +1606,23 @@ private bool PrepareFileAccessMonitoringCommon(LoggingContext loggingContext)
             }
 
             // Untrack the globally untracked paths specified in the configuration 
-            foreach (var path in m_sandboxConfig.GlobalUnsafeUntrackedScopes)
-            {
-                m_fileAccessManifest.AddScope(path, mask: m_excludeReportAccessMask, values: FileAccessPolicy.AllowAll | FileAccessPolicy.AllowRealInputTimestamps);
+            if (m_fileAccessManifest.DirectoryTranslator != null) 

```

Comment:
>(m_fileAccessManifest.DirectoryTranslator [](start = 15, length = 41)

Does this mean that global untracked scopes won't work if there's no directory translator.

Reply:
oh, thanks, I made the logic wrong. Will change it